File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        lst_len = len(nums)
        for i in range(lst_len):
            for j in range(lst_len):
                if (nums[i] + nums[j] == target) and (i != j):
                    return [i, j]

File: 0001-two-sum/0001-two-sum.java
import java.util.Map;
import java.util.HashMap;

class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer, Integer> mapper = new HashMap<Integer, Integer>();
        int numsLen = nums.length;
        for (int i = 0; i < numsLen; i++) {
            int complement = target - nums[i];
            if (mapper.containsKey(complement)) {
                return new int[] {mapper.get(complement), i};
            }
            mapper.put(nums[i], i);
        }
        return new int[] {};
    }
}